config/settings.yml

```diff
@@ -1,7 +1,7 @@
 feature_flags:
   dev_auth:
     local: <%= ENV.fetch("DEV_AUTH_ENABLED", false) %>
-    staging: false
+    staging: <%= ENV.fetch("DEV_AUTH_ENABLED", false) %>
```

Comment:
Is this staging feature flag still needed or with the change in devise initialiser it will never be possible to enable dev_auth on staging (I think as expected to avoid revealing users?)

Reply:
It's still needed as it sets the the dev_auth feature flag to true. The change in the devise initialiser to put the host env local check back in means it can't be enabled on staging

Reply:
Ah sorry yes, understood. Disregard the other comment as I see now these 2 changes go in tandem 👍